### PR TITLE
Comments out wisdom cow, brain trauma events, add timer to meteor event, changes event weights

### PR DESCRIPTION
**Pull request by Azarak**
## Why It's Good For The Game
Wisdom cow is really silly. Brain trauma event is quite common and contributes very little to the round. Meteors were too common for how destructive they are, not a good fit for our long rounds. 

## Changelog
:cl:
tweak: Tweaked some event weights: namely: space pirates, meteors
add: Added a timer to meteors before they arrive, the ETA is alerted to the crew
del: Removed brain trauma and wisdom cow events
/:cl:

### CODE REVIEW

File: code/modules/events/brain_trauma.dm [hunk 1]
```diff
@@ -1,3 +1,5 @@
+//SKYRAT EDIT REMOVAL BEGIN - EVENTS
```
Comment (Gandalf2k15) [inline code/modules/events/brain_trauma.dm]: Let's not comment this out, instead change the possible events to 0. Removing code like this is bad.

File: code/modules/events/meteor_wave.dm [hunk 2]
```diff
@@ -15,6 +15,12 @@
 	var/list/wave_type
 	var/wave_name = "normal"
 
+//SKYRAT EDIT ADDITION - EVENTS
```
Comment (Gandalf2k15) [inline code/modules/events/meteor_wave.dm]: We could have this in a modular file, no?

File: code/modules/events/wisdomcow.dm [hunk 3]
```diff
@@ -1,3 +1,5 @@
+//SKYRAT EDIT REMOVAL BEGIN - EVENTS
```
Comment (Gandalf2k15) [inline code/modules/events/wisdomcow.dm]: Same here, let's not comment it out.

File: code/modules/events/wisdomcow.dm [hunk 4]
```diff
@@ -11,4 +13,5 @@
 	var/turf/targetloc = get_random_station_turf()
 	new /mob/living/simple_animal/cow/wisdom(targetloc)
 	do_smoke(1, targetloc)
-
+*/
+//SKYRAT EDIT REMOVAL END
```
Comment (Gandalf2k15) [inline code/modules/events/wisdomcow.dm]:
```suggestion

```

File: code/modules/events/wisdomcow.dm (hunk 3 above)
Comment (Gandalf2k15) [inline code/modules/events/wisdomcow.dm]:
```suggestion
```

File: code/modules/events/wisdomcow.dm [hunk 5]
```diff
@@ -1,3 +1,5 @@
+//SKYRAT EDIT REMOVAL BEGIN - EVENTS
+/*
```
Comment (Gandalf2k15) [inline code/modules/events/wisdomcow.dm]:
```suggestion
```

File: code/modules/events/wisdomcow.dm [hunk 6]
```diff
@@ -1,3 +1,5 @@
...
 /datum/round_event_control/wisdomcow
 	name = "Wisdom cow"
 	typepath = /datum/round_event/wisdomcow
```
Comment (Gandalf2k15) [inline code/modules/events/wisdomcow.dm:3]: Change max_occurrences to 0. Remember we don't have the dynamic system.

File: code/modules/events/brain_trauma.dm [hunk 7]
```diff
@@ -1,3 +1,5 @@
...
+/*
 /datum/round_event_control/brain_trauma
 	name = "Spontaneous Brain Trauma"
 	typepath = /datum/round_event/brain_trauma
```
Comment (Gandalf2k15) [inline code/modules/events/brain_trauma.dm:3]: Again, change the weight to 0

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]

File: code/modules/events/brain_trauma.dm (hunk 1 above)
Comment (Azarak) [reply code/modules/events/brain_trauma.dm]: How is it bad? It's totally unused, you dont want admins to give a random trauma either.

Comment (Gandalf2k15) [reply code/modules/events/brain_trauma.dm]: It's still part of the game and we shouldn't really be full removing TG stuff, only making it not spawn in game. WE should also be able to trust admins with this

File: modular_skyrat/modules/events/code/modules/events/meteor_wave.dm [hunk 8]
```diff
@@ -0,0 +1,3 @@
+/datum/round_event/meteor_wave/setup()
+	startWhen = rand(90, 180) // Apparently it is by 2 seconds, so 90 is actually 180 seconds, and 180 is 360 seconds. So this is 3-6 minutes
```
Comment (NotRanged) [inline modular_skyrat/modules/events/code/modules/events/meteor_wave.dm:2]: 180 seconds is not enough to order sats and get them up in time at all. Currently even 220 seconds often doesn't get it up in time when cargo isn't perfectly on the ball.

Consider upping the lower limit to 4-4.5 minutes at least.

**Pull request merged**